theia-extensions/viewer-prototype/src/browser/trace-explorer/trace-explorer-sub-widgets/trace-explorer-opened-traces-widget.tsx

```diff
@@ -78,7 +67,8 @@ export class TraceExplorerOpenedTracesWidget extends ReactWidget {
         super.dispose();
         signalManager().off(Signals.EXPERIMENT_OPENED, ({ experiment }) => this.onExperimentOpened(experiment));
         signalManager().off(Signals.EXPERIMENT_CLOSED, ({ experiment }) => this.onExperimentClosed(experiment));
-        signalManager().off(Signals.EXPERIMENT_SELECTED, (experiment: Experiment) => this.onWidgetActivated(experiment));
+        signalManager().off(Signals.EXPERIMENT_SELECTED, (experiment: Experiment) => this.onOpenedTracesWidgetActivated(experiment));
+        signalManager().off(Signals.TRACEVIEWERTAB_ACTIVATED, (experiment: Experiment) => this.onOpenedTracesWidgetActivated(experiment));
```

Comment:
The listener does not get removed because it is not the same instance being passed that was added earlier. (3x)